java/src/jmri/jmrix/tmcc/SerialThrottle.java

```diff
@@ -287,8 +287,9 @@ protected void sendToLayout(int value) {
      * @param Mode ignored, as only 32 is valid
      */
     @Override
-    public void setSpeedStepMode(int Mode) {
-        speedStepMode = 32;
+    public void setSpeedStepMode(jmri.SpeedStepMode Mode) {
+        // TODO(austin): figure out what this is supposed to do and fix it.
+        //speedStepMode = 32;
```

Comment:
I was not able to figure out what the original intent of this code was. `32` as a speed step mode doesn't match any other existing code.